js/tablenator.js

```diff
@@ -0,0 +1,83 @@
+namespacer('bcpl');
+
+bcpl.tablenator = (($, _) => {
+	let $originalTableCache;
+	let breakpoint;
+
+	const buildTwoColumnTables = ($headings, $dataRows) => {
+		const newTableBodyCollection = [];
+
+		if (!$headings || $headings.length === 0 || !$dataRows || $dataRows.length === 0) {
+			return newTableBodyCollection;
+		}
+
+		$dataRows.each((rowIndex, rowElement) => {
```

Comment:
can we move this to it's own function?

Reply:
I thought about that, but it uses variables from the enclosing function. I'd need to wrap this in a function and pass them in again anyway, so I didn't bother.

Reply:
i think its worth breaking it down for testing purposes